src/cinder/gl/GlslProg.cpp

```diff
@@ -137,13 +181,49 @@ void GlslProg::Format::setShaderSource( const std::string &source, std::string *
 
 GlslProg::Format& GlslProg::Format::attrib( geom::Attrib semantic, const std::string &attribName )
 {
-	mAttribSemanticMap[attribName] = semantic;
+    bool exists = false;
```

Comment:
Looks like this was formatted with spaces instead of tabs (you can see in xcode by going to Editor -> Show Invisibles).